src/Sulu/Bundle/WebsiteBundle/Controller/WebsiteController.php

```diff
@@ -161,4 +162,17 @@ protected function getCacheTimeLifeEnhancer(): ?CacheLifetimeEnhancer
 
         return $this->get('sulu_http_cache.cache_lifetime.enhancer');
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedServices()
+    {
+        $subscribedServices = parent::getSubscribedServices();
+        // $subscribedServices['sulu_website.resolver.parameter'] = ParameterResolverInterface::class;
```

Comment:
this method does not affect anything right now.. i guess there is some configuration missing, but this is kind of over my head 🤔